docs/DeveloperGuide.md

```diff
@@ -478,24 +482,49 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
 ### Deleting a student
 
-1. Deleting a student while all students are being shown
+1. Deleting a student while all students are being shown.
 
-   1. Prerequisites: List all students using the `list` command. Multiple students in the list.
+   1. Prerequisite: List all students using the `list` command. There should be **at least 1 student** listed.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   2. Test case: `delete 1`<br>
+      Expected: First contact is deleted from the list. UGTeach displays success message with details of the deleted student.
 
-   1. Test case: `delete 0`<br>
-      Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
+   3. Test case: `delete 0`<br>
+      Expected: No student is deleted. UGTeach displays error message.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+   4. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size)<br>
+      Expected: No student is deleted. UGTeach displays error message.
```

Comment:
Hmm, may I ask if we are gonna standardize on `Similar to previous.` or just repeat the Expected from previous message? I would have to update my PR accordingly too

Reply:
I think we can standardise to similar to previous if doing so does not cause ambiguity

Reply:
I see